core/unix/nudgesig.c

```diff
@@ -50,7 +50,7 @@
 
 /* shared with tools/nudgeunix.c */
 bool
-create_nudge_signal_payload(kernel_siginfo_t *info OUT, uint action_mask,
+create_nudge_signal_payload(kernel_siginfo_t *info OUT, uint action_mask, uint flags,
                             client_id_t client_id, uint64 client_arg)
 {
```

Comment:
Assert that flags is not more than 2 bits.

Reply:
Done.